### ISSUE TITLE
Adding kernel specific metrics to nbresuse

### ISSUE THREAD

**Issue by Gsbreddy** [yuvipanda/nbresuse#40]
Hi All,
I have implemented the solution to capture kernel specific prometheus metrics and would like to contribute back.

Please let me know if it's ok to raise a PR.

Thank you.

**Comment by jtpio** [comment]
Thanks @Gsbreddy.

I guess it sounds similar to what is being discussed in https://github.com/yuvipanda/nbresuse/issues/31?

A PR would be very appreciated as this is a feature many would be interested in! (maybe worth checking the prior work linked in #31 too)

**Comment by Gsbreddy** [comment]
Yes @jtpio , I have referred this and implemented as needed. I am attaching a sample metrics collected. 

[hubmetrics.txt](https://github.com/yuvipanda/nbresuse/files/4655900/hubmetrics.txt)

This contribution just adds per kernel prometheus metrics as shown in file.I will raise a PR if this looks good. Once this is done, I will think of working on some items mentioned in #31.

**Comment by jtpio** [comment]
Nice, thanks @Gsbreddy!

**Comment by Gsbreddy** [comment]
I have raised PR @jtpio

**Comment by jtpio** [comment]
Closing as duplicate of https://github.com/yuvipanda/nbresuse/issues/31
